scripts/core/helpers/uuid.ts

```diff
@@ -8,13 +9,13 @@ export function uuid(a) {
         >> a / 4 // 8 to 11
         ).toString(16) // in hexadecimal
         : ( // or otherwise a concatenated string:
-            [1e7] + // 10000000 +
```

Comment:
Did you really mean to change Array<number> to a string?

Reply:
Yeah it was actually doing `[1e7] + -1e3` which is basically converting both to string and concatenating them. The result was `'10000000-1000'`, the same as `` `${1e7}` + -1e3``. I changed it because the compiler was complaining about types.